EIPS/eip-197.md

```diff
@@ -53,12 +53,11 @@ In order to check that an input is an element of `G_1`, verifying the encoding o
 
 ### Definition of the groups
 
-The groups `G_1` and `G_2` are cyclic groups of prime order `q = 21888242871839275222246405745257275088548364400416034343698204186575808495617` on the elliptic curve `alt_bn128` defined by the curve equation
-`Y^2 = X^3 + 3`.
+The groups `G_1` and `G_2` are cyclic groups of prime order `q = 21888242871839275222246405745257275088548364400416034343698204186575808495617`.
 
-The group `G_1` is a cyclic group on the above curve over the field `F_p` with `p = 21888242871839275222246405745257275088696311157297823662689037894645226208583` with generator `P1 = (1, 2)`.
+The group `G_1` is defined on the curve `Y^2 = X^3 + 3` over the field `F_p` with `p = 21888242871839275222246405745257275088696311157297823662689037894645226208583` with generator `P1 = (1, 2)`.
 
-The group `G_2` is a cyclic group on the same elliptic curve over a different field `F_p^2 = F_p[i] / (i^2 + 1)` (p is the same as above) with generator
+The group `G_2` is defined on the curve `Y^2 = X^3 + 3 ((i+9)^(-1))` over a different field `F_p^2 = F_p[i] / (i^2 + 1)` (p is the same as above) with generator
```

Comment:
Not sure why `Y^2 = X^3 + 3 ((i+9)^(-1))` instead of just `Y^2 = X^3 + 3 / (i+9)`

Reply:
Fixed.